autospec/commitmessage.py

```diff
@@ -168,6 +170,43 @@ def process_NEWS(newsfile):
     return commitmessage, cves
 
 
+def process_git(giturl, oldversion, newversion):
+    """
+    process_git() checks out a git tree and tries to turn the
+    git history into a commit message
+    """
+
+    oldtag = oldversion
+    newtag = newversion
+
+    if len(giturl) < 1:
+        return ""
+    if oldversion == newversion:
+        return ""
+
+    run(["git", "-C", "results", "clone", giturl])
+    p = run(["git", "-C", "results/" + tarball.name, "tag"], stdout=PIPE)
+    tags = p.stdout.decode('utf-8').split('\n')
+
+    for t in tags:
+        i = t.find(oldversion)
```

Comment:
This seems like it could introduce a lot of noise for projects that do version as 1, 2, etc for their versions.

Reply:
is there a "ends-width" thing in python then?